xep-0001.xml

```diff
@@ -295,6 +295,7 @@
 <section1 topic='Discussion Process' anchor='discussion'>
   <p>Once a XEP is published, it becomes available for public discussion within the Standards SIG and the broader XMPP developer community. The XEP author (or Document Shepherd) is responsible for collecting feedback from the XMPP developer community during the life of the XEP and for incorporating such feedback into the proposal. In order to fully participate in discussion of the proposal, they should be subscribed to the Standards list, which is the primary venue for discussion of XMPP Extension Protocols. Changes made based on feedback received by the XEP author (or Document Shepherd) shall be captured in updated versions of the XEP (e.g., 0.2 after 0.1), each of which must be put under source control and subsequently published and announced by the XMPP Extensions Editor.</p>
   <p>If an Experimental XEP is inactive (i.e., no updated versions are published) for a period of twelve (12) months, the XMPP Extensions Editor shall automatically change the status of the XEP to Deferred unless it is in the queue of XEPs under active consideration for advancement by the Approving Body; upon submission of an updated version, the XMPP Extensions Editor shall change the status back to Experimental.</p>
+  <p>Only substantial, non-editorial changes count as <em>activity</em> (or <em>updates</em>) for the purpose of considering moving a XEP from or to Deferred state.</p>
```

Comment:
I suggest tying this into the versioning scheme as that's more concrete (and is consistent with the existing text that only understands substantive updates). e.g. something like 
```suggestion
  <p>Only substantial, non-editorial changes (e.g. those that would cause an updated version of 0.1 to 0.2, not editorial updates from 0.1.1 to 0.1.2) count as <em>activity</em> (or <em>updates</em>) for the purpose of considering moving a XEP from or to Deferred state.</p>
```

Reply:
I thought about that, too, but XEP-0001 does not exactly define how we version things. So to be able to refer to that, we’d have to go ahead and write that down clearly. Which I’m not opposed to, in general, but time and stuff.

Reply:
I think it does, actually, from memory define "Updated Versions" as those that have a 0.x.0 minor version bump, talks about when x.0.0 major bumps happen, and the patch 0.0.x versions aren't mentioned (i.e. out of scope of xep1).

Reply:
@SeveFP I agree that this reads a little odd.